feedback-mailer/feedback-mailer.gs

```diff
@@ -0,0 +1,210 @@
+// Whenever you publish a new version of the add-on, you must increment the below version number. This helps for book-keeping and comparing with previous versions.
+const LAST_PUBLISHED_VERSION = 0;
+
+/*
+Current namedValues object is
+nameValues = {
+  'Timestamp': ['6/7/2015 20:54:13'],
+  'Email Address' : ["efgh@the10xacademy.com"],
```

Comment:
The form has collect email addresses enabled (in the settings) and also it is restricted to 10x domain. (So we cannot change the title of this question)
These are the emails of SSMs who fill the form. We can actually remove "SSM Email ID" question from the form questions, as it is automatically done by the above option.

Limitations:
If the SSM is logged in with bd account or any common account instead of their personal 10x account while filling the form, the above option captures the logged in email account.